.github/workflows/jest.yml

```diff
@@ -0,0 +1,24 @@
+name: Jest
+
+on:
+  workflow_call:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "22"
```

Comment:
node 22 is the latest version, but this package is deployed in legacy environments we should commit to supporting. which versions should we test against? (Github Actions lets you define matrices of test versions)

Reply:
Changed to a matrix